### DISCUSSION TITLE
Dịch feature engineering

### DISCUSSION THREAD

**Discussion by minhduc0711** [mlbvn/handson-ml2-vn#10] [Terminology] [answered]
`feature engineering` -> `xây dựng đặc trưng`
Mọi người thấy ổn không nhỉ?

**Comment by lkhphuc** [comment, marked as answer]
Trong glossary có từ `hand-engineer` là `thiết kế thủ công` rồi, vậy `feature engineer == thiết kế đặc trưng` thì sao?

**Reply by minhduc0711** [reply]
sorry mình bỏ qua mất từ này, vậy chốt là `thiết kế đặc trưng` nhé
